torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -69,32 +20,64 @@ defmodule Torchx.NxDoctestTest do
       Torchx.Backend.__unimplemented__()
       |> Enum.map(fn {fun, arity} -> {fun, arity - 1} end)
       |> Kernel.++(
+        # to_binary - not supported, must call backend_transfer before
         to_binary: 2,
+        # to_batched_list - Shape mismatch
```

Comment:
```suggestion
        # to_batched_list - Shape mismatch
```

I don't think it is shape mismatch but rather the fact we don't support all options. In the Torchx.Backend we have a functionality to raise on unsupported options, we should probably do the same here (and update this comment). :)

Reply:
I ran all tests which weren't commented. For this one, I got:

 shape mismatch in Torchx: expected {2, 2}, got: {1, 2}. Please report this bug

Reply:
Ah, I see. There are 4 tests and only 2 fail.

Reply:
I plan on merging this PR without any changes to functionality or tests, and then adding proper tests for each function in subsequent PRs, after we add Torchx to the CI workflow

Reply:
Sounds good. We can also just add something like "needs investigation". I think the main point is to separate the things we need to look into and the things that are inherently different. :)